src/troute-network/troute/hyfeature_preprocess.py

```diff
@@ -741,7 +742,8 @@ def hyfeature_forcing(
     # Assemble coastal coupling data [WIP]
     #---------------------------------------------------------------------
     # Run if coastal_boundary_depth_df has not already been created:
-    if coastal_boundary_depth_df.empty:
+    #if coastal_boundary_depth_df.empty:
```

Comment:
Should this be un-commented out?

Reply:
It's a temporary solution as for hourly schism data the coastal_boundary_depth_df has to be updated at every time loop, so the lines under the if statement need to be executed at every time loop.  Meanwhile, for multi hourly schism data, the lines need to be executed only once, so the if statement should be there.